streaming/index.js

```diff
@@ -143,7 +144,7 @@ const pgConfigFromEnv = (env) => {
   let baseConfig = {};
 
   if (env.DATABASE_URL) {
-    const parsedUrl = dbUrlToConfig(env.DATABASE_URL);
+    const parsedUrl = pgConnectionString.parse(env.DATABASE_URL);
```

Comment:
Did they release the update? See the fix me PR below

Reply:
No, this has not been merged